.github/workflows/build.yml

```
@@ -348,6 +348,30 @@ jobs:
       - uses: ./.github/actions/download
         with:
           platform: android
+      # Examples のダウンロード
+      - name: Download Examples
+        uses: actions/download-artifact@v4
+        with:
+          pattern: examples_*
+          merge-multiple: false
+      - name: Archive Examples
+        run: |
+          for d in examples_*; do
```

`d` という名前が分かりにくいのでちゃんとした名前を指定して下さい。
また、ワイルドカードで列挙すると一覧性が悪くなるので、以下のようにして下さい。
```suggestion
          for platform in \
            windows_x86_64 \
            macos_arm64 \
            ubuntu-20.04_x86_64 \
            ubuntu-22.04_x86_64 \
            ubuntu-24.04_x86_64 \
            ubuntu-24.04_armv8; do
```

---

`d` を使用することをやめました。